proto/p4/config/v1/p4info.proto

```diff
@@ -65,6 +86,8 @@ message PkgInfo {
   string contact = 7;
   // url for more information, e.g. "http://support.p4.org/ref/p4/switch.p4_v1.0"
   string url = 8;
+  // Annotation map derived from raw annotations
+  map <string,AnnotationBody> anno_map = 9;
```

Comment:
I would actually prefer spelling it out: `annotation_map`

Reply:
OK, will do.